test/javascript/couch_test_runner.js

```diff
@@ -335,6 +340,10 @@ function equals(a,b) {
   }
 }
 
+function notEquals(a,b) {
+  if (a != b) return true;
```

Comment:
It turned out to be okay because in the equal case it returned `undefined`, which is falsy in JS, but I added a clearer return false now.